picard/util/webbrowser2.py

```diff
@@ -23,65 +23,6 @@
 from PyQt5 import QtWidgets
```

Comment:
I can't comment on code above this line. :( But I wanted to note that with the removed code, `os` and `sys` are no longer used, so they also no longer need to be `import`ed here.